reconcile/cli.py

```diff
@@ -265,6 +275,11 @@ def run_integration(func_container, ctx, *args):
         else:
             raise e
 
+    dump_schemas_file = ctx.get('dump_schemas')
+    if dump_schemas_file:
+        with open(dump_schemas_file, 'w') as f:
+            f.write(json.dumps(gql.get_called_schemas()))
+
```

Comment:
Is this temporary? what's next after you have the file?

Reply:
No, it's not temporary. The idea is that if you want to know what schemas an integration depends on you add `--dump-schemas <file>` and that will create that file.

I plan to run all integrations with dry-run mode on the pr check for this repository. And compare the content of each file with the list of schemas defined per integration in the integration schema.